src/storage/v2/disk/storage.cpp

```diff
@@ -738,6 +742,7 @@ VertexAccessor DiskStorage::DiskAccessor::CreateVertex(utils::SkipList<Vertex>::
   if (delta) {
     delta->prev.Set(&*it);
   }
+
```

Comment:
Should we increase number of vertices here? Because I couldn't find code which initializes vertex and edge counters based on stored values on disk.

Reply:
Let's assume that I have 100 vertices on disk and restarted Memgraph.
After restart Memgraph instance I'm running `SHOW STORAGE INFO` and what will I see? 0 or 100?
I think it will be 0, because I haven't created anything.